version.py

```diff
@@ -48,4 +48,4 @@
 #   historical commits sharing the same major version, in which case a
 #   minor version bump suffices.
 
-PROVISION_VERSION = (227, 0)
+PROVISION_VERSION = (228, 0)
```

Comment:
A minor bump should suffice, i.e. `(227, 1)`.

Reply:
Also It looks like @andersk already added Zod as a dependency in this [commit](https://github.com/zulip/zulip/commit/38539c8fed521c6e3f914a0b4709b00e3157db28)!